extensions/wikia/PortableInfobox/tests/PortableInfoboxParserTagControllerTest.php

```diff
@@ -14,6 +14,11 @@ protected function setUp() {
 		$this->controller = new PortableInfoboxParserTagController();
 	}
 
+	protected function tearDown() {
```

Comment:
Please add a comment like in https://github.com/Wikia/app/pull/7467/files#diff-205f19f296b9012bff34f01193484beeR14